src/components/tooltip-popup/styles.scss

```diff
@@ -13,41 +15,49 @@
 
   &__content {
     animation: fadein 700ms ease-out forwards;
+    margin-left: 8px;
 
     display: flex;
     align-items: center;
-    padding: 16px;
+    padding: 8px;
     max-width: 32rem;
     height: 24px;
-    background-color: theme.$color-primary-3;
     border-radius: 8px;
-    font-weight: 600;
-    line-height: 24px;
+    font-weight: 400;
+    line-height: 20px;
+    font-size: 14px;
 
-    /* Dark/Greyscale/Grey-12 */
-    color: theme.$color-greyscale-12;
+    @include glass-shadow-and-blur;
+    @include flat-thick;
+    backdrop-filter: blur(32px);
+    color: $glass-highlight-color;
 
     &[data-side='right'][data-align='end'] {
       border-bottom-left-radius: unset;
     }
+
     &[data-side='right'][data-align='start'] {
       border-top-left-radius: unset;
     }
+
     &[data-side='left'][data-align='end'] {
       border-bottom-right-radius: unset;
     }
+
     &[data-side='left'][data-align='start'] {
       border-top-right-radius: unset;
     }
   }
 
   &__arrow {
-    fill: theme.$color-primary-3;
+    position: absolute;
```

Comment:
I've not run zos on this branch but something seems off if we're using `position: absolute` and minus position values here for the arrow. If the tooltip needs to be in a different position if used elsewhere will the arrow look out of place?

Reply:
yeah could be, i was just trying to fix the css of the current tooltip. Right now, this is the only place it is used (and the tooltip-popup code was copied directly from zUI), so, .. 🤷‍♂️